Formula/t/tree-sitter.rb

```diff
@@ -20,11 +20,15 @@ class TreeSitter < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "a2ede1fcbca769ac6422debbd49fb41ee0af98bfad66153cb58bcd5a10e398ce"
   end
 
+  depends_on "cmake" => :build
   depends_on "rust" => :build
   depends_on "node" => :test
+  depends_on "wasmtime"
 
   def install
-    system "make", "install", "AMALGAMATED=1", "PREFIX=#{prefix}"
+    system "cmake", "-S", "lib", "-B", "build", "-DTREE_SITTER_FEATURE_WASM=ON", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", "lib", "-B", "build", "-DTREE_SITTER_FEATURE_WASM=ON", *std_cmake_args
```

👀 

(Personally, I would hold off on this for at least one more version.)

Reply:
Sure, let's do that.